DEVELOPMENT.md

```diff
@@ -54,10 +54,32 @@ hatch run test
 ```
 
 ## Run integration tests
+- Pre-requisites:
+  - Developer must have a valid AWS account.
+  - Developer must create a Farm, Queue. This can be done easily via Deadline Cloud's AWS Console quick farm create workflow. Please copy the Farm, Queue resource IDs for the next step. Please copy the Queue's Job Attachment S3 bucket name for the next step.
```

Comment:
Good point, I think i can delete the Queue ID as the auto resource creator makes 2 test queues. I was going to use the pre-created one but decided otherwise.

Reply:
But the user must have a queue, which implies the JA bucket being set. I removed the "copy queue id".